examples/audio/eeg_signal.py

```diff
@@ -0,0 +1,527 @@
+                "blink5": "blink",
+                "math1": "math",
+                "math2": "math",
+                "math3": "math",
+                "math4": "math",
+                "math5": "math",
+                "math6": "math",
+                "math7": "math",
+                "math8": "math",
+                "math9": "math",
+                "math10": "math",
+                "math11": "math",
+                "math12": "math",
+                "thinkOfItems-ver1": "thinkOfItems",
+                "thinkOfItems-ver2": "thinkOfItems",
+                "video-ver1": "video",
+                "video-ver2": "video",
+                "thinkOfItemsInstruction-ver1": "thinkOfItemsInstruction",
+                "thinkOfItemsInstruction-ver2": "thinkOfItemsInstruction",
+                "colorRound1-1": "colorRound1",
+                "colorRound1-2": "colorRound1",
+                "colorRound1-3": "colorRound1",
+                "colorRound1-4": "colorRound1",
+                "colorRound1-5": "colorRound1",
+                "colorRound1-6": "colorRound1",
+                "colorRound2-1": "colorRound2",
+                "colorRound2-2": "colorRound2",
+                "colorRound2-3": "colorRound2",
+                "colorRound2-4": "colorRound2",
+                "colorRound2-5": "colorRound2",
+                "colorRound2-6": "colorRound2",
+                "colorRound3-1": "colorRound3",
+                "colorRound3-2": "colorRound3",
+                "colorRound3-3": "colorRound3",
+                "colorRound3-4": "colorRound3",
+                "colorRound3-5": "colorRound3",
+                "colorRound3-6": "colorRound3",
+                "colorRound4-1": "colorRound4",
+                "colorRound4-2": "colorRound4",
+                "colorRound4-3": "colorRound4",
+                "colorRound4-4": "colorRound4",
+                "colorRound4-5": "colorRound4",
+                "colorRound4-6": "colorRound4",
+                "colorRound5-1": "colorRound5",
+                "colorRound5-2": "colorRound5",
+                "colorRound5-3": "colorRound5",
+                "colorRound5-4": "colorRound5",
+                "colorRound5-5": "colorRound5",
+                "colorRound5-6": "colorRound5",
+                "colorInstruction1": "colorInstruction",
+                "colorInstruction2": "colorInstruction",
+                "readyRound1": "readyRound",
+                "readyRound2": "readyRound",
+                "readyRound3": "readyRound",
+                "readyRound4": "readyRound",
+                "readyRound5": "readyRound",
+                "colorRound1": "colorRound",
+                "colorRound2": "colorRound",
+                "colorRound3": "colorRound",
+                "colorRound4": "colorRound",
+                "colorRound5": "colorRound",
+            }
+        },
+        inplace=True,
+    )
+
+
+updateLabel(eeg)
+
+print("After replacing labels")
+print(eeg["label"].unique())
+
+le = LabelEncoder()
```

Comment:
what exactly does LabelEncoder do?  I actually haven't used it before.

Can we get a minor explanation here

Reply:
Well, its more of just a simple one-to-one mapper of sorts to convert any categorical string data into numbers. Something like the following

1 : ObjectA
2 : ObjectB
3 : ObjectC 

and so on. 

Although, in retrospect, as we are converting the labels to one-hot representation later ahead, this does seem a bit redundant. Should I keep it or remove it? It does add a sort of explainability but adds to the computation.

Reply:
It seems that Label-encoding will prove to be essential. I tried removing it and it started throwing incompatibility errors.

Reply:
Totally fine; lets keep it, but just quickly explain that it just constructs a lookup table.